yarn-project/circuit-types/src/tx/tx_receipt.ts

```diff
@@ -11,6 +11,7 @@ export enum TxStatus {
   DROPPED = 'dropped',
   MINED = 'mined',
```

Comment:
Not for this PR but I can't help but think we should rename 'MINED'. Reverted transactions are also 'mined'.

Reply:
Very true. I think they should roughly correspond with the RevertCode options. I added a TODO on https://github.com/AztecProtocol/aztec-packages/issues/4096